api/controllers/service_api/app/completion.py

```diff
@@ -224,6 +241,29 @@ def post(self, app_model: App, end_user: EndUser):
             logger.exception("internal server error.")
             raise InternalServerError()
 
+    def _fetch_workflow_id_by_alias(self, *, app_model: App, workflow_alias: str = "") -> str:
```

Comment:
I suggest removing the default value of `workflow_alias` to make this method simpler and clearer.